src/main/java/com/rtb/projectmanagementtool/user/UserControllerTest.java

```diff
@@ -0,0 +1,56 @@
+package com.rtb.projectmanagementtool.taskblocker;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.dev.LocalDatastoreService;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import java.util.HashSet;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class UserControllerTest {
+    private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(
+          new LocalDatastoreServiceTestConfig()
+          .setAutoIdAllocationPolicy(LocalDatastoreService.AutoIdAllocationPolicy.SEQUENTIAL));
+
+    @Before
+    public void setUp() {
+        helper.setUp();
+    }
+
+    @After
+    public void tearDown() {
+        helper.tearDown();
+    }
+
+    @Test
+    public void testAddUsers() {
+        DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
+        UserController ctl = new UserController();
+        HashSet<String> user1Majors = new HashSet();
+        HashSet<String> user2Majors = newHashSet();
+        user1Majors.add("Biology");
+        user2Majors.add("English");
+        user2Majors.add("Religion");
+        // Add 2 initially
+        ctl.addUserData(
+            ds, /*userID=*/ 1l, /*AuthID=*/ 2l,
+            /*userName=*/ "Sarah", /*userYear=*/ 2022,
+            /*userMajors=*/ user1Majors, /*skills=*/ Skills.LEADERSHIP,
```

Comment:
Skills.LEADERSHIP is what I found to access the enum vals online but doesn't seem to work. Any suggestions?

Reply:
Your test is under the wrong package `package com.rtb.projectmanagementtool.taskblocker;`, try putting in the correct package so it'll find the Skills enum